concourse/client/util.py

```diff
@@ -71,14 +72,22 @@ def has_job_been_triggered(
     job: Job,
```

Comment:
please do not mix two semantics in one method.

firstly: why do we need to wait after all? this is very much worth stating
secondly: retain this method as it was, with the original semantics, which was: if the job was already triggered, return True, otherwise return False; then add a second method, that uses this method to use the first method repeatedly with some waiting involved